### ISSUE TITLE
`svelte/valid-prop-names-in-kit-pages` should handle `form` props also.

### ISSUE THREAD

**Issue by baseballyama** [ota-meshi/eslint-plugin-svelte#344]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.32.0

### What version of `eslint-plugin-svelte` are you using?

2.14.1

### What did you do?

I defined `form` props, and I got `disallow props other than data or errors in Svelte Kit page components.` error.

```svelte
<script lang="ts">
  import type { PageData, ActionData } from './$types';
  export let form: ActionData;
</script>
```

### What did you expect to happen?

Do error at all.

### What actually happened?

`disallow props other than data or errors in Svelte Kit page components.` error came.

### Link to **GitHub Repo** with Minimal Reproducible Example

We can reproduce this on REPL.
https://ota-meshi.github.io/eslint-plugin-svelte/rules/valid-prop-names-in-kit-pages/

### Additional comments

When I read https://kit.svelte.dev/docs/form-actions#anatomy-of-an-action , I realized this specification.
